civihr_default_theme/scss/base/_variables.scss

```diff
@@ -10,38 +14,29 @@ $crm-header-height: 60px !default;
 
 // Grays
 // -----------------------------------------------------------------------------
-$gray-dark:           #555862 !default; // menu bg
-$gray-darker:         #4c4c5d !default; // text darker
-$gray:                #757b8e !default; // main text/lighter, menu item bg hover, footer logo
-$gray-light:          #ecedee !default; // body bg
-$gray-lighter:        #fbfbfc !default; // table odd row
-
-$light:               #fff !default; // well bg, text on dark bg hover
-$light-pale:          #c3cbd6 !default; // text on dark bg
-$light-gray:          #eeeded !default;
+$gray-darker:         #3e3e54 !default;
+$gray-dark:           #464354 !default;
+$gray:                #4d4d69 !default;
+$gray-light:          #9494a5 !default;
+$gray-lighter:        #c2cfd8 !default;
 
-$text-muted:          rgba($gray, 0.6);
+$light:               #fff !default;
+$light-pale:          #e8eef0 !default;
+$light-gray:          #f3f6f7 !default;
 
-$crm-gray-light-alt:  #f0f1f4 !default; // carousel item bg, hr resource bg
-$crm-gray-light-blue: #757b8e !default; // scrollbar thumb, pivot table text
-$crm-gray-darker-alt: #404454 !default; // page title
-$crm-grizzly:         #dcdddd !default;
-$crm-color-neutral:   #949caa !default; // gray shade
-$crm-grayish-blue:    #f4f7f8 !default;
-$crm-gray-matte:      $crm-color-neutral; // disabled font
+$chr-gray-1:          #4e4e78;
```

Comment:
would be better to use an adjective, rather than a number. lighters? darkest? etc